core/templates/hash_map.h

```diff
@@ -240,9 +242,12 @@ class HashMap {
 	}
 
 public:
-	_FORCE_INLINE_ uint32_t get_capacity() const { return hash_table_size_primes[capacity_index]; }
-	_FORCE_INLINE_ uint32_t size() const { return num_elements; }
-
+	_FORCE_INLINE_ uint32_t get_capacity() const {
+		return hash_table_size_primes[capacity_index];
+	}
+	_FORCE_INLINE_ uint32_t size() const {
+		return num_elements;
+	}
 	/* Standard Godot Container API */
 
```

Comment:
I'd keep the separation line at least:
```suggestion
	_FORCE_INLINE_ uint32_t get_capacity() const {
		return hash_table_size_primes[capacity_index];
	}
	_FORCE_INLINE_ uint32_t size() const {
		return num_elements;
	}

	/* Standard Godot Container API */

```

Reply:
This time it was indeed automatically modified by clang-format for some reason. Changed it to what is was before.